.github/workflows/scorecard.yml

```
@@ -30,7 +30,7 @@ jobs:
         with:
           persist-credentials: false
       - name: "Run analysis"
-        uses: ossf/scorecard-action@de386ed459e2f85111697f50fe076d0ea617a32f  # v2.4.1
+        uses: ossf/scorecard-action@18cc9b81307fc5ab3c2cd7092955f06dcfdf8c42  # v2.4.1
```

the latest release is `v2.4.1`
```suggestion
        uses: ossf/scorecard-action@f49aabe0b5af0936a0987cfb85d86b75731b0186  # v2.4.1
```
the commit should be `f49aabe0b5af0936a0987cfb85d86b75731b0186`